zeebe/atomix/cluster/src/test/java/io/atomix/raft/ControllableRaftContexts.java

```diff
@@ -382,12 +383,17 @@ public void snapshotAndCompact(final MemberId memberId) {
   }
 
   public void restart(final MemberId memberId) {
-    raftServers.get(memberId).close();
+    final var raftcontext = raftServers.get(memberId);
+    try (final var ignored = MDC.putCloseable("actor-scheduler", memberId.toString())) {
```

Comment:
🔧 Actor schedule doesn't make much sense for the Raft context 🤔 MAybe we use node id or something similar.